### PR TITLE
Configure code coverage service #4

### PR DESCRIPTION
**Pull request by Dmitry404**
I used cobertura as I'm at least familiar with the tool (on a basic level). I suggest to stay with that 'cause we can always switch to jacoco ~~what a name~~ later.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,3 +12,6 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
+
+after_success:
+- ./gradlew cobertura coveralls
```
Comment (alex-dukhno) [inline .travis.yml]: How about to add a new line?

File: build.gradle [hunk 2]
```diff
@@ -8,3 +12,5 @@ repositories {
 dependencies {
     testCompile 'junit:junit:4.12'
 }
+
+cobertura.coverageFormats = ['html', 'xml']
```
Comment (alex-dukhno) [inline build.gradle]: and here too?

**Review by alex-dukhno — DISMISSED** [review]
Maybe it is better to add a simple test to check that coverage is calculated? Then we can remove it. And also need to check what is the threshold of breaking code coverage level. Last but not least add coverall's badge :smile:

File: .travis.yml (hunk 1 above)
Comment (Dmitry404) [reply .travis.yml]: Why?

File: build.gradle (hunk 2 above)
Comment (Dmitry404) [reply build.gradle]: Why? I removed it for a reason, not accidentally.

**Pull request merged**